x11-terms/terminology/terminology-9999.ebuild

```diff
@@ -1,23 +1,26 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="5"
+EAPI="6"
 
 MY_P=${P/_/-}
 
-if [[ "${PV}" == "9999" ]] ; then
-	EGIT_SUB_PROJECT="apps"
-	EGIT_URI_APPEND="${PN}"
-else
+if [[ "${PV}" != "9999" ]] ; then
 	SRC_URI="https://download.enlightenment.org/rel/apps/${PN}/${MY_P}.tar.xz"
 	KEYWORDS="~amd64 ~x86"
+else
+	EGIT_REPO_URI="https://git.enlightenment.org/apps/${PN}.git"
 fi
 
-inherit enlightenment
+inherit git-r3 meson
```

Comment:
preferably stick the `inherit git-r3` in the `EGIT_blabla` block as well (you can have multiple `inherit` statements)